content/integrations/system.md

```diff
@@ -42,43 +46,26 @@ This check collects the number of CPU cores on a host and CPU times (i.e. system
 ### Setup
 #### Installation
 
-The system_core check is packaged with the Agent, so [install the Agent][4] on any host.
+The system_core check is included in the [Datadog Agent][4] package, so you don't need to install anything else on your server.
 
 #### Configuration
 
-Create a file `system_core.yaml` in the Agent's `conf.d` directory. See the [sample system_core.yaml][5] for all available configuration options:
+1. Edit the `system_core.d/conf.yaml` file in the `conf.d/` folder at the root of your Agent's directory. See the [sample system_core.d/conf.yaml][5] for all available configuration options:  
 
-```
-init_config:
+    ```
+    init_config:
 
-instances:
-  - foo: bar
-```
+    instances:
+        - foo: bar
```

Comment:
if you dont' want to use `foobar`, this has the same effect:
```
instances:
  - {}
```

Reply:
👌